parser/xml_parser.h

```diff
@@ -36,7 +36,7 @@ namespace xml {
 
 class parser {
 public:
-	typedef Common::HashMap<Common::String, tag> tag_format_t;
+	typedef Common::HashMap<Common::String, tag> _tag_formatt;
```

Comment:
This is not correct. Must be `TagFormatT` or not touched at all. It is a type, not a variable.